frontend-project/src/pages/letters/locales/pl-PL.ts

```diff
@@ -49,4 +50,12 @@ export const lettersLocale = {
       createFailed: 'Utworzenie listu nie powiodło się.',
     },
   },
+  files: {
+    errors: {
+      fetchFailed: 'Nie udało się pobrać pliku',
```

Comment:
Kropka na końcu.

Reply:
Da radę żeby to puścić w #908? Nie chcę komplikować już i tak mega długiej PRki i później spinac to z PRką od OAutha.